### ISSUE TITLE
Use simple as default optimization level

### ISSUE THREAD

**Issue by jikuja** [siilisolutions/hedge#72]
https://cljs.github.io/api/compiler-options/optimizations
> :optimizations :simple   ;; <-- recommended for Node

# Why?

* Easier to use external js code without adding proper support for extern files
* Timbre implementation revealed hidden bug with Azure context logger
  ref: https://github.com/siilisolutions/hedge/blob/4e95569cb3e3f9585daaed6e0d8e9f66a3ac6053/library/src/hedge/azure/timbre_appender.cljs#L15-L16

# Does simple optimization profile generate sane output

* Dead code elimination?
* with pretty-print?
* File size
* Some optimizations can be turned on for simple

**Comment by erkkikeranen** [comment]
I disagree, I have encountered problems with simple optimization. Simple does not yield equal results to advanced compiling - simple might stop functions from working.

See #117.
